.eslintrc.js

```diff
@@ -372,9 +372,6 @@ module.exports = {
 				'plugin:jest-dom/recommended',
 				'plugin:testing-library/react',
 			],
-			rules: {
-				'testing-library/no-node-access': 'off',
```

Comment:
Removing this as it's now enabled as part of the `plugin:testing-library/react` recommended rules.